generator/Makefile

```diff
@@ -399,3 +400,10 @@ $(MIBDIR)/readydataos:
 $(MIBDIR)/powercom:
 	@echo ">> Downloading powercom"
 	@curl $(CURL_OPTS) -o $(MIBDIR)/powercom "$(POWERCOM_URL)"
+
+$(MIBDIR)/cisco_imc:
```

Comment:
this target needs to be referenced near line 159 to actually be called during `make mibs`.